exercises/sum.cpp

```diff
@@ -6,3 +6,20 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+using namespace std;
+
+int main() {
+    int n, sum = 0;
+
+    cout << "Enter a positive integer: ";
+    cin >> n;
+
+    for (int i = 1; i <= n; ++i) {
+        sum += i;
+    }
+
+    cout << "Sum = " << sum;
+    return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```